render-candy.js

```diff
@@ -0,0 +1,7 @@
+export function renderCandy(candy) {
+    const li = document.createElement('li');
+    const pEl = document.createElement('p');
+    const headerEl = document.createElement('h2');
+    headerEl.textContent = candy.name;
```

Comment:
I think you forgot to append the li here. There should be a li.append(pEl, headerEl)